user-guide/Troubleshooting/Known_issues/KI_SNMPv3_elements_broken_after_upgrade.md

```diff
@@ -0,0 +1,37 @@
+---
+uid: KI_SNMPv3_elements_broken_after_upgrade
+---
+
+# SNMPv3 elements not loaded correctly after upgrade
+
+## Affected versions
+
+Feature Release versions 10.4.9 to 10.4.12, after an upgrade from a 10.4.0 Main Release version starting from 10.4.0 [CU6].
+
+## Cause
+
+When upgrading from a 10.4.0 Main Release version starting from 10.4.0 [CU6] to Feature Release version 10.4.9 to 10.4.12, DataMiner does not run a required upgrade action, causing SNMPv3 passwords not to be migrated correctly. This causes SNMPv3 connections to lose access to their credentials.
+
+## Fix
+
+No fix is available yet.<!-- RN 41630 -->
+
+## Workaround
+
+After the upgrade, manually force the upgrade action to run by specifying an older DataMiner version than the one currently in use:
+
+1. Stop the DataMiner Agent.
+
+1. On the DMA server, go to the folder `C:\Skyline DataMiner\Upgrades\UpgradeActions\` and run the following command as administrator:
```

Comment:
```suggestion
1. On the DMA server, go to the folder `C:\Skyline DataMiner\Upgrades\UpgradeActions\MigrateSNMPv3PasswordsToElementConfig` and run the following command as administrator:
```
The full path of the executable is "C:\Skyline DataMiner\Upgrades\UpgradeActions\MigrateSNMPv3PasswordsToElementConfig\MigrateSNMPv3PasswordsToElementConfig.exe"

I'm now also realizing that it is possible to run the command without navigating to that folder if you specify the full path and surround it with double quotes:
`"C:\Skyline DataMiner\Upgrades\UpgradeActions\MigrateSNMPv3PasswordsToElementConfig\MigrateSNMPv3PasswordsToElementConfig.exe" /oldversion:10.3.0 /newversion:10.5.0`
This could then be copy pasted in a Command Prompt regardless of the current directory.